src/sentry/static/sentry/app/stores/environmentStore.jsx

```diff
@@ -32,6 +37,9 @@ const EnvironmentStore = Reflux.createStore({
       },
     }));
     this.trigger(this.items, 'initial');
+
+    // Update the default environment in the latest context store
+    setDefaultEnvironment(this.getDefault());
```

Comment:
I think we should avoid stores triggering actions, but I can't of a clean way to do this with reflux